fastlane/Fastfile

```diff
@@ -771,6 +771,44 @@ lane :copyright do
   )
 end
 
+desc 'Get frameworks size'
+lane :get_frameworks_size do
+  root_dir = "Build/SDKSize"
+  archive_dir = "#{root_dir}/DemoApp.xcarchive"
+
+  # Archive the demo app
+  xcodebuild(
+    archive: true,
+    archive_path: archive_dir,
+    scheme: "DemoApp"
+  )
+
+  # Export with ad-hoc and thinning support to be able
+  # to get the correct thinned size of Assets.car
+  xcodebuild(
+    archive_path: archive_dir,
+    export_options_plist: "Scripts/SDKSizeExportOptions.plist",
+    export_archive: "#{root_dir}/Export",
+    export_path: "#{root_dir}/Export"
+  )
+
+  packaging_url = "../#{root_dir}/Export/Packaging.log"
+  packaging_file_content = File.read(packaging_url)
+  match = packaging_file_content.match(/\b(\d+)\sbytes\sfor\s\.\/Payload\/ChatSample\.app\/Frameworks\/StreamChatUI\.framework\/Assets\.car\b/)
+  assets_thinned_size = match[1].to_i
```

Comment:
At the moment I did not find a better way to get the Assets.car with the correct size. Using it from the Products folder directly, it is not the correct size since the assets are not thinned. So we need to simulate an ad-hoc export with App thinning enabled. Then, the real size is located in the Packaging.log, but I'm not sure if there's an easier way to do this TBH.

For now, this seems to work, and we have the size reports similar to what Emerge Tools report.